app/bundles/CoreBundle/Twig/Helper/ButtonHelper.php

```diff
@@ -257,14 +261,15 @@ public function setLocation($location)
     /**
      * Reset the buttons.
      *
-     * @param        $buttonCount
+     * @param string $buttonCount
      * @param string $groupType
      * @param null   $item
      *
      * @return $this
      */
     public function reset(Request $request, $buttonCount, $groupType = self::TYPE_GROUP, $item = null)
     {
+        //@escopecz: I think there is a possible bug here
         $this->location       = $buttonCount;
```

Comment:
@escopecz the value that is stored in the location is `$buttonCount`, while we have ` setLocation` function, so this line is either necessary or could cause a bug that is not tested, because we are passing the wrong value

Reply:
Do you have some steps to reproduce the bug? If not then TBH I don't want to get deep into this considering other priorities.